opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/ServerTracingFilter.java

```diff
@@ -59,7 +57,8 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
                 spanBuilder.asChildOf(extractedSpanContext);
             }
 
-            Span span = spanBuilder.start();
+            Span span = spanBuilder.startManual();
+            tracer.makeActive(span);
```

Comment:
@malafeev exactly, I think 3. is the most consistent solution. If we cannot deactivate span then do not activate. I'm wondering what other people think about this.